examples/graphql-subscriptions/graphql_subscriptions.metatags

```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
-keywords: ballerina, ballerina by example, bbe, graphql, subscription
+description: This example demonstrates defining GraphQL subscription operations.
```

Comment:
```suggestion
description: This example demonstrates defining GraphQL subscription operation.
```

Reply:
Updated as: `This example demonstrates defining a GraphQL subscription operation.`.